dockerfiles/jdk17/rocky/is/Dockerfile

```diff
@@ -52,48 +66,48 @@ RUN set -eux; \
 ENV JAVA_HOME=/opt/java/openjdk \
     PATH="/opt/java/openjdk/bin:$PATH"
 
-# Set Docker image build arguments.
-# Build arguments for user/group configurations.
+# set Docker image build arguments
```

Comment:
Comment should being with a capital letter and end with a fullstop. Check other places as well.